addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -17,7 +17,7 @@
 
 .o_ComposerTextInput_textarea {
     padding: 10px;
-    max-height: 200px;
+    max-height: 100px;
```

Comment:
I tried but I don't find it's pretty, it's kind of mindfucking that the scroll impacts something that is not "glued" to it.
I also can't manage to avoid this kind of glitch without scrolling also the buttons (and so they can disappear somehow).

![image](https://user-images.githubusercontent.com/48204996/90006071-8273fa80-dc98-11ea-8ac5-7e2fae4ae16d.png)

Reply:
> Just wondering, do we want the rule even if not compact? Wouldn't that be annoying in some other cases (chatter)?

We could indeed increase that limit in not compact situations even if it does not bother me.
![image](https://user-images.githubusercontent.com/48204996/90008037-067bb180-dc9c-11ea-95c4-2c395b99adca.png)

What do you suggest ? 200px ?

Reply:
I guess 200px is fine when not compact since it was the previous value in that case.

Reply:
> I tried but I don't find it's pretty, it's kind of mindfucking that the scroll impacts something that is not "glued" to it.
> I also can't manage to avoid this kind of glitch without scrolling also the buttons (and so they can disappear somehow).
> 
> ![image](https://user-images.githubusercontent.com/48204996/90006071-8273fa80-dc98-11ea-8ac5-7e2fae4ae16d.png)

If we manage to fix the glitch with the buttons I think it will be less confusing.

Reply:
Any clue of how to fix that ? However I make it, the height is limited to the container scroll height ... (tried relative + absolute, flex stretch, height 100%, .. )